protocol/communities/manager.go

```diff
@@ -1355,12 +1475,42 @@ func (m *Manager) DeleteCategory(request *requests.DeleteCommunityCategory) (*Co
 	return changes.Community, changes, nil
 }
 
-func (m *Manager) HandleCommunityDescriptionMessage(signer *ecdsa.PublicKey, description *protobuf.CommunityDescription, payload []byte, shard *common.Shard) (*CommunityResponse, error) {
+func (m *Manager) Queue(signer *ecdsa.PublicKey, community *Community, clock uint64, payload []byte) error {
+
+	m.logger.Info("queuing community", zap.String("id", community.IDString()), zap.String("signer", common.PubkeyToHex(signer)))
+
+	communityToValidate := communityToValidate{
+		id:         community.ID(),
+		clock:      clock,
+		payload:    payload,
+		validateAt: uint64(time.Now().UnixNano()),
+		signer:     crypto.CompressPubkey(signer),
+	}
+	err := m.persistence.SaveCommunityToValidate(communityToValidate)
+	if err != nil {
+		m.logger.Error("failed to save community", zap.Error(err))
+		return err
+	}
+
+	return nil
+}
+
+func (m *Manager) HandleCommunityDescriptionMessage(signer *ecdsa.PublicKey, description *protobuf.CommunityDescription, payload []byte, shard *common.Shard, verifiedOwner *ecdsa.PublicKey) (*CommunityResponse, error) {
```

Comment:
please, double-check who is calling this fn...
I have a crash in one of my tests in `handleChatMessage`, `m.handleWrappedCommunityDescriptionMessage` returns nil due to a queue